lsp-mode.el

```diff
@@ -7006,7 +7006,7 @@ SESSION is the active session."
         (list :processId nil
               :rootPath (lsp-file-local-name (expand-file-name root))
               :clientInfo (list :name "emacs"
-                                :version (emacs-version))
+                                :version (string-trim emacs-version))
```

Comment:
I think this should be

```
(string-trim (emacs-version))
```

You are missing curly brackets around `emacs-version`.

Reply:
I'm sorry I'm not really good at LISP :)
```
              :clientInfo (list :name "emacs"
                                :version (string-trim (emacs-version)))
```

Doesn't work here though, the resulting string is the original one :(

Reply:
`(emacs-version)` and `emacs-version` yield different results.